server/middlewares/stats.js

```diff
@@ -71,13 +110,27 @@ const trackTxSubmissions = mung.json((body, req, res) => {
         : 'otherTxSubmissions'
     const txSubmissionSuccess = body.Right ? 'successful' : 'unsuccessful'
 
-    incrCountersBy(`${txSubmissionType}:${txSubmissionSuccess}`, 1)
-
-    if (txSubmissionSuccess === 'successful') {
-      const {txBody} = req.body
-      const txOutAmount = parseTxBodyOutAmount(txBody)
-
-      incrCountersBy(`${txSubmissionType}:sentOut`, txOutAmount)
+    try {
+      if (txSubmissionSuccess === 'successful') {
+        const {txBody} = req.body
+        const txOutAmount = parseTxBodyOutAmount(txBody)
+
+        trackEvent({
+          eventCategory: 'Payments',
+          eventAction: `${txSubmissionType}:sentOutSuccess`,
+          eventLabel: 'successful payment',
+          eventValue: txOutAmount,
+        })
+      } else {
+        trackEvent({
+          eventCategory: 'Payments',
+          eventAction: `${txSubmissionType}:sentOutFail`,
+          eventLabel: 'unsuccessful payment',
+        })
+      }
+    } catch (err) {
+      // eslint-disable-next-line no-console
+      console.error(`Tracking event failed - ${err.message}`)
```

Comment:
I would rather log the whole error payload. The error may not always have the message property